README.md

```diff
@@ -620,6 +636,95 @@ $ ./bin/swctl checkHealth --grpcTLS=true
 
 </details>
 
+<details>
+<summary>Output manifest of OAP server</summary>
+
+Output manifest with default custom resource
+
+```shell
+./bin/swctl install manifest oap
+```
+
+Load overlay custom resource from flag
+```shell
+swctl install manifest ui -f oap-cr.yaml
+```
+
+Load overlay custom resource from stdin
+```shell
+cat ui-cr.yaml | ./bin/swctl install manifest oap -f=-
+```
+
+Apply directly to Kubernetes
+```shell
+./bin/swctl install manifest oap -f oap-cr.yaml | kubectl apply -f-
```

Comment:
`-f-` need a space or `=` between `-f` and `-`?

Reply:
Sadly for the CLI lib that we're using, space or `=` seems not redundant. But it doesn't when I use cobra. I didn't dive into why this happened.